src/data/_contributors.yaml

```diff
@@ -900,6 +900,17 @@ petelepage:
   lanyrd: petele
   description:
     en: Pete is a Developer Advocate
+ 
+ kennith:
```

Comment:
There's an extra space at the beginning of ` kennith:` that is breaking the yaml processor.